finesse/gui/temp_control.py

```diff
@@ -71,15 +66,15 @@ def _create_controls(self) -> QGridLayout:
         self._btns["cold"].clicked.connect(
             partial(self._toggle_axis_visibility, name="cold")
         )
+        self._btns["hot"].setMaximumWidth(80)
+        self._btns["cold"].setMaximumWidth(80)
```

Comment:
I think a cleaner way would be to just prevent them from expanding, via:
`self._btns["hot"].setSizePolicy(QSizePolicy.Policy.Fixed, QSizePolicy.Policy.Fixed)
self._btns["cold"].setSizePolicy(QSizePolicy.Policy.Fixed, QSizePolicy.Policy.Fixed)`

Reply:
I tried this, but the problem is that the buttons will have different width, as the text inside has different width. The result looks slightly annoying with the two buttons a bit different.